package.json

```diff
@@ -17,6 +17,7 @@
     "decompress-zip": "^0.3.0",
     "defaults": "^1.0.3",
     "detect-indent": "^5.0.0",
+    "fastreplacestring": "git+https://github.com/IwanKaramazow/FastReplaceString.git",
```

Comment:
@IwanKaramazow stylistic nit, but some of your indentation is funky. Looks like perhaps mixed spaces and tabs?